app/serializers/api/admin/commodities/commodity_csv_serializer.rb

```diff
@@ -0,0 +1,54 @@
+require_relative '../../../concerns/api/v2/csv/csv_serializer'
+
+module Api
+  module Admin
+    module Commodities
+      class CommodityCsvSerializer
+        include Api::V2::Csv::CsvSerializer
+
+        column :goods_nomenclature_sid, column_name: 'SID' do |row|
```

Comment:
Why do you need the block?

Reply:
because inside the block, we are calling
`row[:goods_nomenclature_sid]`
and not
`row.goods_nomenclature_sid`

In other words, the attributes are not present as methods (they come from the `all_commodities` query)